CMakeLists.txt

```diff
@@ -50,6 +50,10 @@ else()
     set(BUILD_ILMBASE_STATIC OFF)
 endif()
 
+if (NOT OPENEXR_BUILD_SHARED)
```

Comment:
This should be 

````
if (OPENEXR_BUILD_STATIC OR NOT OPENEXR_BUILD_SHARED)
````

Reply:
Ah, gotcha. On Windows, for the tests especially, I'd favor static. That said, that's probably a tiny extra bit of logic for a future commit.